src/pages/workspace/workflows/WorkspaceWorkflowsPage.tsx

```diff
@@ -110,42 +137,64 @@ function WorkspaceWorkflowsPage({policy, betas, route}: WorkspaceWorkflowsPagePr
                 title: translate('workflowsPage.makeOrTrackPaymentsTitle'),
                 subtitle: translate('workflowsPage.makeOrTrackPaymentsDescription'),
                 onToggle: () => {
-                    // TODO will be done in https://github.com/Expensify/Expensify/issues/368335
+                    const isActive = policy?.reimbursementChoice === CONST.POLICY.REIMBURSEMENT_CHOICES.REIMBURSEMENT_MANUAL;
```

Comment:
It should be the opposite, no? The toggle should be on when the type is not manual
```suggestion
                    const isActive = policy?.reimbursementChoice !== CONST.POLICY.REIMBURSEMENT_CHOICES.REIMBURSEMENT_MANUAL;
```